code/game/objects/structures/crates_lockers/closets/secure/chaplain.dm

```diff
@@ -34,6 +34,3 @@
 	new /obj/item/weapon/cell/small(src)
 	new /obj/item/weapon/tool/sword/nt/longsword(src)
 	new /obj/item/weapon/tool/knife/dagger/nt(src)
-	new /obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee(src)
```

Comment:
You need to make individual design disks for each weapon. Also, the design disks for the NT Dagger and NT Shortsword should start off roundstart next to the biolathe.

Reply:
Done.